svcs/amigo/amigo.go

```diff
@@ -721,6 +724,16 @@ func JSONEndpoint(next http.HandlerFunc) http.HandlerFunc {
 		next(w, r)
 	}
 }
+
```

Comment:
I understood it, we have already discussed.However, for safety and guideline, could you add brief explanation for the function (to see it on go docs)  by starting with "getParentChallengeTag ...."?